cmd/goose/driver_mysql.go

```diff
@@ -69,7 +69,8 @@ func registerTLSConfig(pemfile string, sslcert string, sslkey string) error {
 		clientCert := make([]tls.Certificate, 0, 1)
 		certs, err := tls.LoadX509KeyPair(sslcert, sslkey)
 		if err != nil {
-			log.Fatal(err)
+			log.Fatalf("Error in load x509 keypair: %v", err)
```

Comment:
`Fatalf` calls `os.Exit` under the hood.

We should try and bubble up all errors to the caller, in this case, you can either:

```
if err != nil {
  return err
}
```

Or if you want to annotate the error do

```
return fmt.Errorf("failed to load x509 keypair: %w")
```

I know there are existing places `log.Fatal` is called but we'd like to avoid this pattern moving forward and eventually clean this up.

Reply:
Makes sense. Done. 

I have followed 

```
return fmt.Errorf("failed to load x509 keypair: %w", err)
```